packages/data-stores/src/site/actions.ts

```diff
@@ -238,6 +245,9 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 			if ( 'site_vertical_id' in settings ) {
 				yield receiveSiteVerticalId( siteId, settings.site_vertical_id );
 			}
+			if ( 'launchpad_screen' in settings ) {
+				yield receiveLaunchpadScreen( siteId, settings.launchpad_screen );
+			}
```

Comment:
+1. We don't need this due to the catch-all call below it.

Reply:
Addressed!